spec/unleash/client_spec.rb

```diff
@@ -96,29 +96,31 @@
 
     features_response_body = '{
       "version": 1,
-      "features": {
-        "name": "toggleName",
-        "enabled": true,
-        "strategies": [{ "name": "default" }],
-        "variants": [
-          {
-            "name": "a",
-            "weight": 50,
-            "payload": {
-              "type": "string",
-              "value": ""
+      "features": [
+        {
+          "name": "toggleName",
+          "enabled": true,
+          "strategies": [{ "name": "default" }],
+          "variants": [
+            {
+              "name": "a",
+              "weight": 50,
+              "payload": {
+                "type": "string",
+                "value": ""
+              }
+            },
+            {
+              "name": "b",
+              "weight": 50,
+              "payload": {
+                "type": "string",
+                "value": ""
+              }
```

Comment:
The existing tests weren't accurate with the actual response the server provides. The server provides an array of features, but the test used to have it returning a hash -- in the code there's places where we iterate over all the features, but this was just iterating over the key/value pairs of the hash and were just coincidentally passing.